File: VentasVB/frmLogin.vb
Public Class frmLogin
    Dim con As Conexion = New Conexion()
    Private Sub btnLogin_Click(sender As Object, e As EventArgs) Handles btnLogin.Click
        If con.login(tbCorreo.Text, tbContraseña.Text) Then
            Dim menu As frmVentas = New frmVentas()
            menu.Show()
            Hide()
        Else
            MessageBox.Show("Usuario o contraseña incorrectas")
        End If
    End Sub
End Class

File: Ventas/frmLogin.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Ventas
{
	public partial class frmLogin : Form
	{
		Conexion con = new Conexion();

		public frmLogin()
		{
			InitializeComponent();
		}

		private void btnLogin_Click(object sender, EventArgs e)
		{
			if (con.login(tbCorreo.Text, tbContraseña.Text))
			{
				frmVenta menu = new frmVenta();
				menu.Show();
				Hide();
			}
			else MessageBox.Show("Usuario o contraseña incorrectas");
		}
	}
}
